### PR TITLE
Task2.3

### PR DESCRIPTION
**Pull request by fujoshik**

### CODE REVIEW

File: Task 2/Task2.3/Trainings/Interfaces/ITraining.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Task2._3.Trainings.Interfaces
+{
```
Comment (IgorKheidorov) [inline Task 2/Task2.3/Trainings/Interfaces/ITraining.cs]: Interface without methods?

File: Task 2/Task2.3/Trainings/Training.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
+﻿using Task2._3.Trainings.Interfaces;
+
+namespace Task2._3.Trainings
+{
+    public class Training : BaseTraining
+    {
+        private int _trainingsCount;
```
Comment (IgorKheidorov) [inline Task 2/Task2.3/Trainings/Training.cs:5]: BaseTraining[] will be ok, but not perfect.\
The right structure is Training : TrainingEnity (->Description inside)
PracticalLesson: BaseTraing: TrainingEntity

And all entities implements IClonable interface

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: Task 2/Task2.3/Trainings/Interfaces/ITraining.cs (hunk 1 above)
Comment (fujoshik) [reply Task 2/Task2.3/Trainings/Interfaces/ITraining.cs]: Fixed

File: Task 2/Task2.3/Trainings/Training.cs (hunk 2 above)
Comment (fujoshik) [reply Task 2/Task2.3/Trainings/Training.cs:5]: Fixed

**Pull request merged**